code/__defines/gamemode.dm

```diff
@@ -51,9 +51,18 @@
 #define MODE_VAMPIRE "vampire"
 #define MODE_THRALL "thrall"
 
-
 #define DEFAULT_TELECRYSTAL_AMOUNT 25
 
+// Defines (NOT FLAGS) for making secret, random, and mixed secret less snowflake-string
+// and number dependant.
+#define ROUNDTYPE_SECRET       1
+#define ROUNDTYPE_MIXED_SECRET 2
+#define ROUNDTYPE_RANDOM       3
```

Comment:
Since these are used in hide_mode. I wonder if `ROUNDTYPE_RANDOM` should be flipped to 0 🤔